### DISCUSSION TITLE
How to change date display format from `mm/dd/yyyy` to `dd/mm/yyyy` for DateInput

### DISCUSSION THREAD

**Discussion by upender7270** [adobe/react-spectrum#6947] [Q&A] [answered]
I'm developing a date range picker using react-aria-components, how can I change the date display format from `mm/dd/yyyy` to `dd/mm/yyyy` for DateInput

**Comment by snowystinger** [comment]
On my phone or I'd go get a link. But you should be able to find some previous issues/discussions about this topic.

**Reply by upender7270** [reply]
I have gone through the issues and discussion before adding this discussion.

**Reply by snowystinger** [reply, marked as answer]
Here's a couple I found:
https://github.com/adobe/react-spectrum/discussions/3734
https://github.com/adobe/react-spectrum/issues/6266

We also describe how the date is formatted here: https://react-spectrum.adobe.com/react-aria/DatePicker.html#international-calendars
> Dates are automatically displayed in the appropriate calendar system for the user's locale. The calendar system can be overridden using the [Unicode calendar locale extension](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/calendar#adding_a_calendar_in_the_locale_string), passed to the [I18nProvider](https://react-spectrum.adobe.com/react-aria/I18nProvider.html) component.
